src/components/MissionsList.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Good job! Avoid using in the future this type of linter disable unless extremely necessary,